packages/components/src/components/ScalarIcon/icons/index.ts

```diff
@@ -1,16 +1,21 @@
+import { defineAsyncComponent } from 'vue'
+
 import { type ICONS } from './icons'
 
-const icons = import.meta.glob('./*.svg', { as: 'raw', eager: true })
+const icons = import.meta.glob<SVGElement>('./*.svg')
```

Comment:
If this uses the SVG loader could we just do this?
```suggestion
const icons = import.meta.glob<SVGElement>('./*.svg?component')
```
https://www.npmjs.com/package/vite-svg-loader#component

Reply:
Would they still be lazy loaded then?

Reply:
Or would that load all of them at once